public/components/navbar.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN"
+      crossorigin="anonymous"
+    />
+    <style>
+      a {
+        color: black;
+        text-decoration: none;
+      }
+    </style>
+    <script src="./js/home.js"></script>
+    <title>Home Page</title>
+  </head>
+  <body>
+    <nav class="navbar navbar-expand border border-dark">
+      <div class="container-fluid">
+        <div class="collapse navbar-collapse">
+          <ul class="navbar-nav me-auto">
+            <li class="nav-item nav-link">
```

Comment:
I don't think there's any need for these "Test" links. We can get rid of them. Just keep "Cart"

Reply:
Or actually, "Test1" could be "Home" and just redirects to the home page `href="/"`. But get rid of the others